docs/Delivery-Practices/Build-and-Release/Engineering/Peer-Reviewing/Release-Plan-Review.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Release Plan Review
+authors: 
+  - Rhodri Hewitson
+reviewed: 
+reviewer:
+next*review: 2023-06-01
+---
+# Release Plan Review
+
+## Why do we have release documentation? 
+Until we have fully automated our release process, as engineers we must fill out a release plan to make sure care and consideration is taken to ensure we reduce downtime as much as possible and release the BLI as safely as possible,
```

Comment:
As this documentation lives in the Playbook, it applies to all platforms - so perhaps we should differentiate this slightly from platforms that will have a fully automated process (e.g. Access Management - and others as we roll it out!).

I'd also be tempted to reword the next bit slightly to account for all team members. So perhaps the whole sentence could read:

> For any platforms without a fully automated release process, delivery teams are required to create a release plan, and make sure care and consideration is taken in reducing downtime and releasing the BLI as safely as possible.

Reply:
> The 'Why' section doesn't read right to me but not sure the best way to rephrase. Maybe something like ...
> 
> Until we have fully automated our release process, as Engineers we must fill out a release plan to make sure care and consideration is taken in reducing downtime and releasing the BLI as safely as possible.

Yeah it was clumsily written. I've stolen Dave's suggestion since it mentions delivery teams which is better than just mentioning engineers .